ideascube/conf/base.py

```diff
@@ -315,6 +315,6 @@
 
 
 SESSION_COOKIE_AGE = 60 * 60  # Members must be logged out after one hour
-BACKUP_FORMAT = 'zip'  # One of 'zip', 'tar', 'bztar', 'gztar'
+BACKUP_FORMAT = 'gztar'  # One of 'tar', 'bztar', 'gztar'
```

Comment:
Since we have Python 3.4, we should also be able to support xz compression.

I know, this is entirely unrelated to your pull-request, just figured I'd mention it. :stuck_out_tongue_winking_eye: